static/js/components/CommentList.jsx

```diff
@@ -19,6 +19,19 @@ import UserAvatar from "./UserAvatar";
 dayjs.extend(relativeTime);
 dayjs.extend(utc);
 
+export const shortenFilename = (filename) => {
+  if (filename.length <= 15) {
+    return filename;
+  }
+  const extensionLength = filename.split(".", 2)[1].length;
```

Comment:
What if there are no dots in the filename for some reason?

Reply:
Good point. Case implemented